olive/passes/onnx/vitis_ai_quantization.py

```diff
@@ -129,39 +140,66 @@
     "quant_format": PassConfigParam(
         type_=str,
         default_value="QDQ",
-        searchable_values=Categorical(["QDQ"]),
+        searchable_values=Categorical(["QDQ", "QOperator"]),
         description="""
             QDQ format quantize the model by inserting QuantizeLinear/DeQuantizeLinear on the tensor.
         """,
     ),
+    "need_layer_fusing": PassConfigParam(
+        type_=bool,
+        default_value=False,
+        searchable_values=Boolean(),
+        description="""
+            Perform layer fusion for conv-relu type operations
+        """,
+    ),
+    "execution_providers": PassConfigParam(
```

Comment:
the accelerator_spec on the run has the execution provider info. Is this one needed?